### ISSUE TITLE
Python Renderer Compatibility

### ISSUE THREAD

**Issue by olls** [itsapi/pycraft#131]
Are we going to continue to maintain the Python renderer, if not should we remove it?

**Comment by TiberiumPY** [comment]
I think it's better to have Python renderer, so new contributors can understand rendering system more easily.

**Comment by olls** [comment]
Yes, that's a good reason. At the moment the C renderer in #134 is probably easier to understand than the Python renderer. The Python renderer is an inefficient mess of spaghetti code at the moment and would not help anyone understand the rendering, so if we keep it we need to rewrite it to match the operation of the new C renderer (which is probably the correct answer...).

#130 is a good description about how the new lighting rendering system now works if you are interested. We should probably write up the rest of the renderer as well into our imaginary documentation!

**Comment by olls** [comment]
I vote for removal of the Python renderer at this point.  There is no point in rewriting to match the C renderer just for the sake of it.  And it's no use to anyone in its current state.
